CHANGES.txt

```diff
@@ -4,8 +4,11 @@ Changes
 2.6 (unreleased)
 ----------------
 
-- Nothing changed yet.
-
+- compatibility for python 3
+- update code to work with python2 and python3
+- adjust egg to work with newer version of setuptools
```

Comment:
(Augh, I forgot this was already merged so *of course* it would be a separate change.  Early morning no :coffee: yet.)